src/node.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Entry point for setting up of Requests Debugger Tool.
+ * Initiates actions such as processing of args, setting up loggers,
+ * initiating all connectivity checks and stats collection before starting
+ * the proxy tool.
+ */
+
+var constants = require('../config/constants');
+var LogFiles = constants.LOGS;
+var RdGlobalConfig = constants.RdGlobalConfig;
+var CommandLineManager = require('./commandLine');
+var ConnectivityChecker = require('./connectivity');
+var RdHandler = require('./server');
+var StatsFactory = require('./stats/statsFactory');
+var LogManager = require('./logger');
+var fs = require('fs');
+var path = require('path');
+var Utils = require('./utils');
+
+var RdTool = {
+
+  /**
+   * Initializes the Logging directory, Loggers & Stats Handlers.
+   */
+  initLoggers: function () {
+    var basePath = RdGlobalConfig.logsPath ? path.resolve(RdGlobalConfig.logsPath) : process.cwd();
+    RdGlobalConfig.LOGS_DIRECTORY = path.resolve(basePath, 'RequestsDebuggerLogs');
```

Comment:
`DELETE_EXISTING_LOGS` is a boolean set using the CLI arg `--del-logs` which will delete any existing log files of that name in that directory to initialize new ones. `LOGS_DIRECTORY` is the path where the log files will be created.

its because, it could be an absolute path, i.e. starting with `/some/absolute/path` or a relative path to the current working directory, i.e. `its/a/relative/path`. Thus adding a check over the value provided by the user helps in detecting the correct path.

Reply:
Resolving. The concern was around using camelCase and constants inconsistency. Its fixed.